src/applications/mhv-landing-page/utilities/data/data.unit.spec.js

```diff
@@ -65,5 +65,98 @@ describe(manifest.appName, () => {
         expect(resolvedLink.href).to.equal('/new');
       });
     });
+
+    describe('countUnreadMessages', () => {
+      it('unread count only uses inbox and custom folders', () => {
+        // Custom folder Ids are > 0
+        function customFolderId() {
+          return Math.round(Math.random() * 99999) + 1;
```

Comment:
Typically, we want deterministic test suites with non-random input. Instead of a random id number, just hard code one.

Reply:
Makes sense and makes it simpler since the code only cares the ID is positive. Done.